Examples/Tests/nuclear_fusion/analysis_proton_boron_fusion.py

```diff
@@ -80,7 +80,7 @@
 
 ## Checks whether this is the 2D or the 3D test
 with open("./warpx_used_inputs") as warpx_used_inputs:
-    is_2D = re.search("geometry.dims\s*=\s*2", warpx_used_inputs.read())
+    is_2D = re.search(r"geometry.dims\s*=\s*2", warpx_used_inputs.read())
```

Comment:
Here I think it would be also safe to remove the `\s*`, since `warpx_used_inputs` is generated automatically and uses only one white space, as in `key = value`, but up to you. It might make it more readable overall:
```suggestion
    is_2D = re.search("geometry.dims = 2", warpx_used_inputs.read())
```